objects/src/accounts/code.rs

```diff
@@ -157,8 +157,8 @@ impl Eq for AccountCode {}
 
 impl Serializable for AccountCode {
     fn write_into<W: ByteWriter>(&self, target: &mut W) {
+        // module imports and source locations are not serialized on account code
```

Comment:
I would write this as "debug info (this includes module imports and source locations) is not serialized with account code"

Reply:
changed!